snekbox/nsjail.py

```diff
@@ -129,16 +137,24 @@ def _consume_stdout(self, nsjail: subprocess.Popen) -> str:
         return "".join(output)
 
     def python3(
-        self, code: str, *, nsjail_args: Iterable[str] = (), py_args: Iterable[str] = ("-c",)
-    ) -> CompletedProcess:
+        self,
+        py_args: Iterable[str],
+        files: Iterable[FileAttachment] = (),
+        *,
+        nsjail_args: Iterable[str] = (),
+    ) -> EvalResult:
         """
         Execute Python 3 code in an isolated environment and return the completed process.
 
         The `nsjail_args` passed will be used to override the values in the NsJail config.
         These arguments are only options for NsJail; they do not affect Python's arguments.
 
         `py_args` are arguments to pass to the Python subprocess before the code,
-        which is the last argument. By default, it's "-c", which executes the code given.
+        which is the last argument.
+
+        If `use_file` is True, `code` will be written to a file the last argument will be
+        the file name `main.py`. If False, `code` will be passed as the last argument.
+        The default of None will use a file unless `c` is in `py_args`.
```

Comment:
This seems out of date.

Reply:
Updated in [f12f8f3](https://github.com/python-discord/snekbox/pull/159/commits/f12f8f3915620cf5f20ef5c36e488517a4e73e4b)